samples/package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@google-cloud/dlp": "^3.0.2",
     "@google-cloud/pubsub": "^2.0.0",
+    "google-auth-library": "^6.0.6",
```

Comment:
You shouldn't need to use this directly.  Curious - where did it get used?

Reply:
I wanted to test using it to avoid re-instantiating another client in our test suites.

Reply:
Instantiating a new client in the test suite is perfectly ok!  We do this in other places. I think bringing this in as a dependency will create some confusion.